mingw-w64-itk: fails to rebuild

```
CMake Error at D:/M/msys64/clang64/lib/cmake/vtk/VTK-vtk-module-find-packages.cmake:115 (find_package):
    By not providing "FindQt6.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "Qt6", but
    CMake did not find one.
  
    Could not find a package configuration file provided by "Qt6" (requested
    version 6.4) with any of the following names:
  
      Qt6Config.cmake
      qt6-config.cmake
  
    Add the installation prefix of "Qt6" to CMAKE_PREFIX_PATH or set "Qt6_DIR"
    to a directory containing one of the above files.  If "Qt6" provides a
    separate development package or SDK, be sure it has been installed.
  Call Stack (most recent call first):
    D:/M/msys64/clang64/lib/cmake/vtk/vtk-config.cmake:152 (include)
    Modules/Bridge/VtkGlue/itk-module-init.cmake:9 (find_package)
    CMake/ITKModuleEnablement.cmake:338 (include)
    CMakeLists.txt:542 (include)
  
  
  -- Configuring incomplete, errors occurred!
```

--- a/mingw-w64-gdcm/0009-link-to-iconv.patch
+++ b/mingw-w64-gdcm/0009-link-to-iconv.patch
@@ -1,14 +1,11 @@
 --- a/Source/MediaStorageAndFileFormat/CMakeLists.txt
 +++ b/Source/MediaStorageAndFileFormat/CMakeLists.txt
-@@ -207,6 +207,11 @@
- if(GDCM_USE_JPEGLS)
-   target_link_libraries(gdcmMSFF LINK_PRIVATE ${GDCM_CHARLS_LIBRARIES})
+@@ -228,7 +228,7 @@
+ if(GDCM_USE_SYSTEM_JSON)
+   target_link_libraries(gdcmMSFF LINK_PRIVATE ${JSON_LIBRARIES})
  endif()
-+
-+if(NOT MSVC)
-+  find_package(Iconv REQUIRED)
-+  target_link_libraries(gdcmMSFF PUBLIC Iconv::Iconv)
-+endif()
- 
- if(CMAKE_COMPILER_IS_GNUCXX AND MINGW)
-   # I am getting:
+-if(UNIX)
++if(UNIX OR MINGW)
+   find_package(Iconv)
+   target_link_libraries(gdcmMSFF LINK_PRIVATE ${Iconv_LIBRARIES})
+ endif()

--- a/mingw-w64-gdcm/PKGBUILD
+++ b/mingw-w64-gdcm/PKGBUILD
@@ -4,7 +4,7 @@ _realname=gdcm
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=3.0.19
-pkgrel=1
+pkgrel=2
 pkgdesc="The Grassroots DICOM library (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
@@ -43,20 +43,36 @@ sha256sums=('52ee78a70aaee7e179d584c1b7116da607dbc59905c833dadbd85944c301769a'
             '008635edc8c200b2adf8adf33c3439d39706e9b7ed0e644d7d0ebe18d9a020eb'
             'ed9c4020bd39cadf4d78c72aa4be5afaf3d0a743406e8bee6c5936299e5d1d33'
             'c2222761e9a2b7572a553d6af22dfd87c3a1ff53e8d876b8c26f47436cf54218'
-            '531cc02810335cdae4efa9452beb2e1d211c0822e431ca006e5ad5e2ef69de92')
+            '6cd51dcc48fe587f655fdcb4ba1b15adff8b95a97a943d94b59d6cb85143e0e9')
+
+apply_patch_with_msg() {
+  for _patch in "$@"
+  do
+    msg2 "Applying $_patch"
+    patch -Nbp1 -i "${srcdir}/$_patch"
+  done
+}
 
 prepare() {
   cd "${srcdir}/${_realname}-${pkgver}"
-  patch -p1 -i ${srcdir}/0002-cmake-config-destination.patch
-  patch -p1 -i ${srcdir}/0004-fix-find-openssl.patch
-  patch -p1 -i ${srcdir}/0005-fix-cross-initializing.patch
-  patch -p1 -i ${srcdir}/0006-openjpeg2-visibility.patch
-  patch -p1 -i ${srcdir}/0007-getopt-global-var.patch
-  patch -p1 -i ${srcdir}/0008-no-posix_memalign-on-MINGW.patch
-  patch -p1 -i ${srcdir}/0009-link-to-iconv.patch
+  apply_patch_with_msg \
+    0002-cmake-config-destination.patch \
+    0004-fix-find-openssl.patch \
+    0005-fix-cross-initializing.patch \
+    0006-openjpeg2-visibility.patch \
+    0007-getopt-global-var.patch \
+    0008-no-posix_memalign-on-MINGW.patch \
+    0009-link-to-iconv.patch
 }
 
 build() {
+  declare -a _extra_config
+  if check_option "debug" "n"; then
+    _extra_config+=("-DCMAKE_BUILD_TYPE=Release")
+  else
+    _extra_config+=("-DCMAKE_BUILD_TYPE=Debug")
+  fi
+
   echo "==> Build static version"
   [[ -d "${srcdir}/build-static-${MSYSTEM}" ]] && rm -rf "${srcdir}/build-static-${MSYSTEM}"
   mkdir -p "${srcdir}/build-static-${MSYSTEM}" && cd "${srcdir}/build-static-${MSYSTEM}"
@@ -67,7 +83,7 @@ build() {
     -G"Ninja" \
     -DCMAKE_CXX_STANDARD=17 \
     -DCMAKE_INSTALL_PREFIX="${MINGW_PREFIX}" \
-    -DCMAKE_BUILD_TYPE=Release \
+    ${_extra_config[@]} \
     -DCMAKE_SYSTEM_PREFIX_PATH="${MINGW_PREFIX}" \
     -DGDCM_BUILD_SHARED_LIBS=OFF \
     -DGDCM_USE_SYSTEM_ZLIB=ON \
@@ -108,7 +124,7 @@ build() {
     -G"Ninja" \
     -DCMAKE_CXX_STANDARD=17 \
     -DCMAKE_INSTALL_PREFIX="${MINGW_PREFIX}" \
-    -DCMAKE_BUILD_TYPE=Release \
+    ${_extra_config[@]} \
     -DCMAKE_SYSTEM_PREFIX_PATH="${MINGW_PREFIX}" \
     -DGDCM_BUILD_SHARED_LIBS=ON \
     -DGDCM_USE_SYSTEM_ZLIB=ON \
@@ -142,8 +158,8 @@ build() {
 
 package() {
   cd "${srcdir}/build-static-${MSYSTEM}"
-  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --build ./ --target install
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --install .
 
   cd "${srcdir}/build-shared-${MSYSTEM}"
-  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --build ./ --target install
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --install .
 }

--- a/mingw-w64-itk/PKGBUILD
+++ b/mingw-w64-itk/PKGBUILD
@@ -4,7 +4,7 @@ _realname=itk
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=5.2.1
-pkgrel=5
+pkgrel=6
 pkgdesc='An open-source C++ toolkit for medical image processing (mingw-w64)'
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
@@ -39,11 +39,12 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
              "${MINGW_PACKAGE_PREFIX}-libtheora"
              "${MINGW_PACKAGE_PREFIX}-openslide"
              "${MINGW_PACKAGE_PREFIX}-openturns"
+             "${MINGW_PACKAGE_PREFIX}-openvdb"
              "${MINGW_PACKAGE_PREFIX}-openvr"
              "${MINGW_PACKAGE_PREFIX}-pdal"
              "${MINGW_PACKAGE_PREFIX}-proj"
-             "${MINGW_PACKAGE_PREFIX}-qt5-base"
-             "${MINGW_PACKAGE_PREFIX}-qt5-declarative"
+             "${MINGW_PACKAGE_PREFIX}-qt6-base"
+             "${MINGW_PACKAGE_PREFIX}-qt6-declarative"
              "${MINGW_PACKAGE_PREFIX}-unixodbc"
              "${MINGW_PACKAGE_PREFIX}-utf8cpp")
 optdepends=("${MINGW_PACKAGE_PREFIX}-opencv: ITK-OpenCV bridge"
@@ -74,13 +75,6 @@ build() {
   mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"
 
   local -a _extra_config
-  if [[ "$CARCH" = 'i686' ]]; then
-    _extra_config=('-DVNL_CONFIG_ENABLE_SSE2_ROUNDING=OFF')
-  fi
-  if [[ "$CARCH" = 'x86_64' ]]; then
-    _extra_config=('-DVNL_CONFIG_ENABLE_SSE2_ROUNDING=ON')
-  fi
-
   if check_option "debug" "n"; then
     _extra_config+=("-DCMAKE_BUILD_TYPE=Release")
   else
@@ -134,6 +128,7 @@ build() {
     -DITK_SKIP_PATH_LENGTH_CHECKS=1 \
     -DITK_FORBID_DOWNLOADS=ON \
     -DVNL_CONFIG_THREAD_SAFE=ON \
+    -DVNL_CONFIG_ENABLE_SSE2_ROUNDING=ON \
     "${_extra_config[@]}" \
     "../InsightToolkit-${pkgver}"
 

